zulip_bots/zulip_bots/bots/converter/test_converter.py

```diff
@@ -5,8 +5,8 @@
 class TestConverterBot(StubBotTestCase):
     bot_name = "converter"
 
-    def test_bot(self):
-        dialog = [
+    def test_bot(self) -> None:
+        expected = [
             ("", 'Too few arguments given. Enter `@convert help` '
                  'for help on using the converter.\n'),
             ("foo bar", 'Too few arguments given. Enter `@convert help` '
```

Comment:
You forgot to change `dialog` to `expected` at the end of this test, which is why tests are failing. Also, I recommend running `./tools/test-bots` and `./tools/run-mypy` before pushing your commits! :)